### DISCUSSION TITLE
New Sections - Default layout

### DISCUSSION THREAD

**Discussion by Andreln** [thuliteio/doks#362] [Q&A]
Hello.

I am a bit new to Hugo, but I have gotten this theme up and running and I've started customizing it to my liking.

I just have a quick question about the default layout.  Creating new folders and markdown files in the content folder does not seem to work as I thought it would as the page is created, but no content is showed on the page. 

Should they not use the "_default" layout?

**Comment by epogrebnyak** [comment]
Probably depends were you create the new files - the 'documentation' folder?

**Reply by epogrebnyak** [reply]
Maybe this helps: https://github.com/h-enk/doks/discussions/62

**Reply by Andreln** [reply]
Thank you for the reply.

I tried creating new pages in the content folder, not `content/doks`.
I was under the impression that new content. i.e. `content/new_page/_index.md` would inherit default layout?
